htdocs/expedition/list.php

```diff
@@ -473,7 +501,7 @@
 
 $arrayofmassactions = array(
 	'builddoc' => img_picto('', 'pdf', 'class="pictofixedwidth"').$langs->trans("PDFMerge"),
-	//'classifyclose'=>$langs->trans("Close"), TODO massive close shipment ie: when truck is charged
+	'classifyclose'=>$langs->trans("Close"), TODO massive close shipment ie: when truck is charged
```

Comment:
missing // before TODO

Reply:
Hi @frederic34 That's right.

Thank you for fix.

I'm still begineer.

Reply:
@frederic34 
I think "TODO massive close shipment ie: when truck is charged" should remove.. right?